src/dotnet/commands/dotnet-migrate/Program.cs

```diff
@@ -80,13 +80,8 @@ public static int Run(string[] args)
             }
             catch (Exception ex)
```

Comment:
i feel your concern - do you know what kind of exceptions can we get here? i can send this last commit as a separate PR